components/toast/toast.css

```diff
@@ -0,0 +1,84 @@
+
+.practical .btn-box button{
+    background-color: var(--secondary-color);
+    box-shadow: rgba(60, 64, 67, 0.3) 0px 1px 2px 0px, rgba(60, 64, 67, 0.15) 0px 1px 3px 1px;
+    color: var(--primary-color);
+    border: none;
+    cursor: pointer;
+    padding: 1rem;
+    border: 1px solid var(--black-color);
+    margin-right: 1rem;
+}
+
+#baseline-toast{
```

Comment:
changes: use classes for styling, avoid using id's for styling

Reply:
Noted